### PR TITLE
Update labels in `contributing.md`

### LINKED ISSUE #193: Update contributing.md to reflect label rename

### PR DESCRIPTION
**Pull request by its-miroma**
Fixes #193

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
   },
   "devDependencies": {
     "@types/glob": "^8.1.0",
-    "@types/node": "^22.5.5",
+    "@types/node": "^22.7.5",
```
Comment (modmuss50) [inline package.json]: Was there a reason to update the depedencies?
Shouldnt the package-lock file also changed?

Comment (its-miroma) [reply package.json]: I had to add `@vueuse/core`, because `vitepress dev` failed without it.

Comment (its-miroma) [reply package.json]: I downgraded all other dependencies

Comment (modmuss50) [reply package.json]: You should be using `npm run dev`, Im always very hesitent to add new depedencies like this. I will take a closer look.

Comment (modmuss50) [reply package.json]: That works for me, I dont see why this package should be added.

Comment (its-miroma) [reply package.json]: `@vueuse/core` is being used here: https://github.com/FabricMC/fabric-docs/blob/main/.vitepress/theme/components/BannerComponent.vue#L2

Comment (its-miroma) [reply package.json]: `npm run dev` does not fail, but opening the page in the browser does. I'll attach a screenshot

Comment (modmuss50) [reply package.json]: As you can see from the package lock changes `@vueuse/core` was already in there as vitepress depends on it. I dont think this PR makes any functional changes to how it works.

Comment (its-miroma) [reply package.json]: Honestly I can't reproduce the problem I had yesterday, so I guess it's fixed (:

Comment (modmuss50) [reply package.json]: Humm, do let me know if that changes. Sorry about being so cautious when adding dependepes, especially npm ones :D

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**